jekyll/_cci2/create-an-orb.adoc

```diff
@@ -134,17 +131,17 @@ You will be provided with a link to the project building on CircleCI where you c
 
 Using the orb development kit makes use of link:/docs/dynamic-config/[dynamic configuration], you will need to enable this feature. You will receive an error on your first pipeline that will state that this feature is not yet enabled.
 
-Following the link:/docs/dynamic-config/#getting-started-with-dynamic-config-in-circleci[Getting started with dynamic config in CircleCI] guide, open the **Project Settings** page for your orb on CircleCI, navigate to the **Advanced** tab, and click on the **Enable dynamic config using setup workflows** button.
+Following the link:/docs/dynamic-config/#getting-started-with-dynamic-config-in-circleci[getting started with dynamic config in CircleCI] guide, open the **Project Settings** page for your orb on CircleCI, navigate to the **Advanced** tab, and click on the **Enable dynamic config using setup workflows** button.
 
-Once enabled, all future commits to your project will run through the full pipeline and test your orb. We could manually re-run the pipeline at this point, but since we are only working with sample code at this moment, we don't need to.
+Once enabled, all future commits to your project will run through the full pipeline and test your orb. You could manually re-run the pipeline at this point, but since you are only working with sample code at this moment, it is not necessary.
 
 === 9. Develop your orb
 
 From a non-default branch (you will be moved to the `alpha` branch automatically at setup), begin modifying the sample orb code to fit your requirements. On each _push_, your orb will be automatically built and tested. More information on developing your orb can be found on the link:/docs/orb-author/#writing-your-orb[orb authoring process] page.
 
 Be sure to view the link:https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/test-deploy.yml[.circleci/test-deploy] file to view how your orb components are being tested, and modify your tests as you change your orb. Learn more about testing your orb on the link:/docs/testing-orbs/[orb testing methodologies] page.
 
-When you are ready to deploy the first production version of your orb, find information on deploying changes on the link:/docs/creating-orbs/[Orb Publishing Process] page.
+When you are ready to deploy the first production version of your orb, find information on deploying changes on the link:/docs/creating-orbs/[orb publishing process] page.
 
 video::kTeRJrwxShI[youtube]
```

Comment:
I don't know if we can do anything about this, but it's confusing that the video order is 1, 3, 2. Was the process recorded out of order? Again, not really something we can probably control, but just a weird observation.

Reply:
Yes I thought the same. The content and both came from CPE originally with the videos in this order. I'll leave as is for now but can always come back to this.